account_financial_risk/tests/test_account_financial_risk.py

```diff
@@ -121,7 +123,8 @@ def test_invoices(self):
         invoice2 = self.invoice.copy({"partner_id": self.invoice_address.id})
         self.assertAlmostEqual(self.partner.risk_invoice_draft, 550.0)
         self.assertAlmostEqual(self.partner.risk_invoice_unpaid, 550.0)
-        wiz_dic = invoice2.post()
+        invoice_form = Form(invoice2)
+        wiz_dic = invoice_form.post()
```

Comment:
This is a `Form` and can't run methods. I think the problem is that you are posting twice, so the solution is:

```suggestion
```